contracts/IdleTokenFungible.sol

```diff
@@ -463,9 +452,10 @@ contract IdleTokenFungible is Initializable, ERC20Upgradeable, ReentrancyGuardUp
       _updateFeeInfo();
       uint256 _unclaimedFees = unclaimedFees;
       if (_unclaimedFees > 0) {
-        // send fees
-        _mint(feeAddress, _unclaimedFees * ONE_18 / _tokenPrice());
-        // reset fee counter
+        // send fees (lastNAV was just updated in _updateFeeInfo)
+        _mint(feeAddress, _unclaimedFees * totalSupply() / lastNAV);
+        // reset fee counter and update lastNAV
+        lastNAV += _unclaimedFees;
```

Comment:
Is this line here main causes of the bug?

Reply:
not only this but mostly because fees that were yet to be included in unclaimedFees were not excluded from lastNAV